targets/csharp/source/PlayFabSDK/source/PluginContract.cs

```diff
@@ -3,6 +3,7 @@
     public enum PluginContract
     {
         PlayFab_Serializer,
-        PlayFab_Transport
+        PlayFab_Transport,
+        PlayFab_PollyTransport
```

Comment:
As mentioned offline, I'd prefer to not add an additional contract type (since it would be misaligned with the other languages). Instead, I'd prefer we add comments and possibly deprecation notes around the instanceName to avoid future confusion.